components/FeedbackModal.js

```diff
@@ -7,20 +7,21 @@ import ErrorAlert from './ErrorAlert'
 import api from '../lib/api-client'
 import { CreatableDropdown } from './Dropdown'
 import { ClearButton } from './IconButton'
+import useTurnstileToken from '../hooks/useTurnstileToken'
 
 export default function FeedbackModal() {
   const [text, setText] = useState(null)
   const [submitting, setSubmitting] = useState(false)
-  const [turnstileToken, setTurnstileToken] = useState(null)
   const [error, setError] = useState(null)
+  const [isOpen, setIsOpen] = useState(false)
   const { accessToken } = useContext(UserContext)
   const [formData, setFormData] = useReducer((state, action) => {
     if (action.type === 'reset') return {}
     if (action.type === 'prefill') return action.payload
     return { ...state, [action.type]: action.payload }
   }, {})
+  const { turnstileToken, turnstileContainerRef } = useTurnstileToken('feedback')
```

Comment:
I was actually suggesting the parent component manage the ref and pass it to the hook. Then the parent component could conditionally render the container using some state var. Would this pattern make the code simpler?